example/src/simple/component/master.js

```diff
@@ -0,0 +1,140 @@
+/**
+ *
+ * @module component/master
+ * @author Greg Babula [greg.babula@mlb.com]
+ * @desc entry point for all component specific functionality
+ * jQuery and Bootstrap provided for example, however neither are required
```

Comment:
description needs to be updated, we dont use `jQuery` and `Bootstrap` in examples anymor .